pinot-query-runtime/src/test/java/org/apache/pinot/query/runtime/executor/OpChainSchedulerServiceTest.java

```diff
@@ -144,6 +144,7 @@ public void shouldScheduleOpChainEvenIfNoOpChainsInAWhile()
     OpChain opChain = getChain(_operatorA);
     CountDownLatch latch = new CountDownLatch(3);
     AtomicBoolean returnedOpChain = new AtomicBoolean(false);
+    Mockito.when(_operatorA.nextBlock()).thenReturn(TransferableBlockUtils.getNoOpTransferableBlock());
```

Comment:
Added this because otherwise we were getting a silent NullPointerException logged in the UTs